### DISCUSSION TITLE
Collect .epub files with parsing issues here

### DISCUSSION THREAD

**Discussion by aoout** [aoout/obsidian-epub-importer#48] [General]
Just like the title says.

**Comment by aoout** [comment]
[冠军早餐.zip](https://github.com/aoout/obsidian-epub-importer/files/14882763/default.zip)
The error that occurred on this .epub is obvious.

**Reply by HarshIceHummer** [reply]
I think we cannot upload epub files to GitHub

**Reply by aoout** [reply]
> I think we cannot upload epub files to GitHub

We can change the suffix to .zip first, and then we can upload it.

**Comment by aoout** [comment]
[愉悦回路.zip](https://github.com/aoout/obsidian-epub-importer/files/15062541/default.zip)

**Comment by aoout** [comment]
[知识的边界.zip](https://github.com/aoout/obsidian-epub-importer/files/15494040/default.zip)

**Comment by HarshIceHummer** [comment]
[Beginning Python Games Development - Harrison Kinsley.epub.zip](https://github.com/user-attachments/files/18116581/Beginning.Python.Games.Development.-.Harrison.Kinsley.epub.zip)

Epub Importer is not importing this epub.

**Reply by aoout** [reply]
epubimporter is not recommended for reading programming books.

**Comment by HarshIceHummer** [comment]
[Linux Shell Programming_ Pocket Primer - Oswald Campesato.epub.zip](https://github.com/user-attachments/files/18116935/Linux.Shell.Programming_.Pocket.Primer.-.Oswald.Campesato.epub.zip)

**Reply by aoout** [reply]
epubimporter is not recommended for reading programming books.

**Comment by HarshIceHummer** [comment]
[Cyberlibertarianism_ The Right-Wing Politi - David Golumbia.epub.zip](https://github.com/user-attachments/files/18117191/Cyberlibertarianism_.The.Right-Wing.Politi.-.David.Golumbia.epub.zip)

**Reply by aoout** [reply]
Already fixed.

**Comment by HarshIceHummer** [comment]
[J. R. R. Tolkien_ a biography - Humphrey Carpenter.epub.zip](https://github.com/user-attachments/files/18117203/J.R.R.Tolkien_.a.biography.-.Humphrey.Carpenter.epub.zip)

**Reply by aoout** [reply]
Already fixed.

**Comment by HarshIceHummer** [comment]
[Certified ethical hacker - Wikipedia.epub.zip](https://github.com/user-attachments/files/18139075/Certified.ethical.hacker.-.Wikipedia.epub.zip)

**Reply by aoout** [reply]
Already fixed.

**Comment by HarshIceHummer** [comment]
[Three Truths of Well Being - Sadhguru.epub.zip](https://github.com/user-attachments/files/18165909/Three.Truths.of.Well.Being.-.Sadhguru.epub.zip)

**Reply by aoout** [reply]
I can import this book. Can you describe your situation in more detail?

**Reply by HarshIceHummer** [reply]
![image](https://github.com/user-attachments/assets/fc8c6b78-6853-40eb-9af6-6dccf9b20bf9)
Even though all these files have been created, there is no text in any of them i.e. these are all empty notes as shown below -
![image](https://github.com/user-attachments/assets/cee7375f-d7d3-438c-9bab-5f1e68e7dabe)
As a result, I am copy-pasting the text for each note from the epub file.

**Reply by aoout** [reply]
Already fixed.

**Reply by HarshIceHummer** [reply]
Is it possible that I could be facing these issues (even though already fixed) because I'm on Linux OS?

**Reply by aoout** [reply]
Sorry, I know nothing about linux. Is the result on 0.6.5 exactly the same as before? Still all blank pages? I'd like more details.

**Reply by aoout** [reply]
I only use Windows myself. I have no way of debugging on Linux.

**Comment by HarshIceHummer** [comment]
[The Art of Computer Programming, Volume 1_ - Donald E. Knuth.epub.zip](https://github.com/user-attachments/files/18387020/The.Art.of.Computer.Programming.Volume.1_.-.Donald.E.Knuth.epub.zip)

I understand that this plugin shouldn't be used on programming books, but for me, books are books and I'm just adding them to my Vault using this plugin.

**Reply by aoout** [reply]
I can add it to my vault on 0.6.6.

**Reply by HarshIceHummer** [reply]
I am able to add it as well using 0.6.6

**Comment by HarshIceHummer** [comment]
[One hundred years of solitude - Gabriel Garcia Marquez.epub.zip](https://github.com/user-attachments/files/18482281/One.hundred.years.of.solitude.-.Gabriel.Garcia.Marquez.epub.zip)

Not able to import this one using 0.6.6
